ServiceBase/src/main/java/com/cafebazaar/servicebase/Utils.kt

```diff
@@ -0,0 +1,26 @@
+package com.cafebazaar.servicebase
```

Comment:
Change file name to packageHelper or something like this, util is very general

Reply:
Done.